.travis.yml

```diff
@@ -1,5 +1,5 @@
-sudo: true
-
+os: linux
+dist: xenial
```

Comment:
Please elaborate as I am not too familiar with travis. Why was this change made?

Reply:
Travis was producing warnings, If not defined, travis assumed these values, but better to make it explicit.

Reply:
thanks